www/class/centreonHost.class.php

```diff
@@ -1014,7 +1011,8 @@ public function setTemplates($hostId, $templates = array(), $remaining = array()
         $i = 1;
         $queryValues = array();
         foreach ($templates as $templateId) {
-            if (!isset($templateId)
+            if (
+                !isset($templateId)
```

Comment:
```suggestion
                ! isset($templateId)
```

Reply:
legacy code out of php-cs-fixer scope I think, but fixed ✅